vm-systemd/qubes-sysctl-minimal-sys-net.service

```diff
@@ -0,0 +1,19 @@
+[Unit]
+Description=Apply minimal sys-net kernel variables
+DefaultDependencies=no
+Conflicts=shutdown.target
+After=systemd-modules-load.service qubes-sysinit.service
+Before=sysinit.target shutdown.target
+ConditionPathIsReadWrite=/proc/sys/net/
+ConditionPathExists=/var/run/qubes-service/minimal-netvm
+ConditionPathExists=/var/run/qubes/this-is-netvm
```

Comment:
What is the intended effect? Right now it enables it when _both_ conditions are true - so, if it's netvm _and_ the minimal-netvm service is enabled. If you meant to automatically enable it in netvm but also in potentially other if requested, then it should have `|` prefix. But otherwise, I'd prefer a single condition - either just "minimal-netvm" service, or automatic in an netvm.